interactions/utils/utils.py

```diff
@@ -231,3 +245,66 @@ def disable_components(
             for _components in components:
                 for component in _components.components:
                     component.disabled = True
+
+
+def get_channel_history(
+    http: "HTTPClient",
```

Comment:
Can you make it so either the http client or the bot can be passed? Many users probably won't even know what the http client is, which is intended due to the underscore prefix